build-webhook.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -e
+
+pack build europe-west3-docker.pkg.dev/sap-se-gcp-istio-dev/public/kpack-667/webhook --builder gcr.io/cf-build-service-public/ci/tiny-builder -e BP_GO_TARGETS=./cmd/controller --publish --trust-builder
```

Comment:
could the the tag argument to `pack` be parameterized in this script? also `pack` the args have changed since this to 

```
-B gcr.io/cf-build-service-public/ci/kpack-builder -e BP_GO_TARGETS=./cmd/controller -e BP_BUILD_LIBGIT2=true -e BP_GO_BUILD_FLAGS="-tags=\"static\""  --publish
```

You could also just remove the scripts if you don't want to update them.